db/migrate/20180302125803_create_saving_goals.rb

```diff
@@ -0,0 +1,9 @@
+class CreateSavingGoals < ActiveRecord::Migration[5.1]
+  def change
+    create_table :saving_goals do |t|
+      t.integer :amount
```

Comment:
Whenever you're dealing with money in an application it's probably best to make use of `decimals` instead of `integer` value. - Just a note for future reference

Reply:
This is interesting because we thought so too but Magnus suggested we should use integers, and that you use cents instead of euro's, which solves the decimal issue. So you put 1000, and you're actually talking about 10 euros. He sent us this: 
https://stackoverflow.com/questions/3730019/why-not-use-double-or-float-to-represent-currency